plugins/vendor/zoom/lib/model/oAuth/kZoomOauth.php

```diff
@@ -63,12 +69,20 @@ private static function curlRetrieveTokensData($url, $userPwd, $header, $postFie
 
 	/**
 	 * set two minutes off the token expiration, avoid 401 response from zoom
-	 * @param int $expiresIn
-	 * @return int
+	 * @param array $tokenData
 	 */
-	public static function getValidUntil($expiresIn)
+	public static function setTokenExpiryAbsoluteTime(&$tokenData)
```

Comment:
Thinking to remove the call by reference since we're only using 1 value
I'll make the code a little ugly like:
$tokenData[self::EXPIRE_IN] = setTokenExpiryAbsoluteTime($tokenData[self::EXPIRE_IN]);

What do you think?